clear-env-mig.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Deletes existing cases from DynamoDB and ElasticSearch
+
+# Usage
+#   ./clear-env.sh $ENV
+
+# Requirements
+#   - terraform must be installed on your machine
+#   - aws cli must be installed on your machine
+#   - aws credentials must be setup on your machine
+#   - node must be setup on your machine
+
+# Arguments
+#   - $1 - the environment to clear
+
+[ -z "$1" ] && echo "The branch name to check must be provided as the \$1 argument." && exit 1
+[ -z "${USTC_ADMIN_PASS}" ] && echo "You must have USTC_ADMIN_PASS set in your environment" && exit 1
+[ -z "${AWS_ACCESS_KEY_ID}" ] && echo "You must have AWS_ACCESS_KEY_ID set in your environment" && exit 1
+[ -z "${AWS_SECRET_ACCESS_KEY}" ] && echo "You must have AWS_SECRET_ACCESS_KEY set in your environment" && exit 1
+
+ENV=$1
+
+$(which terraform) > /dev/null
+if [[ "$?" == "1" ]]; then
+  echo "Terraform was not found on your path. Please install terraform."
+  exit 1
+fi
+
+./web-api/clear-elasticsearch-index.sh $ENV
+./web-api/setup-elasticsearch-index.sh $ENV
+
+pushd web-api
+node clear-dynamodb-table.js $ENV
```

Comment:
@ericsorenson -- another question, as this clears out the DynamoDB table, would this also create orphaned Cognito users that would have been associated with the user records in DynamoDB? If that's correct, is there any way to handle them?